src/commons/application/Application.tsx

```diff
@@ -138,6 +144,8 @@ const Application: React.FC<ApplicationProps> = props => {
             <Route path="/playground" component={Playground} />
             <Route path="/contributors" component={Contributors} />
             <Route path="/sourcecast/:sourcecastId?" component={SourcecastContainer} />
+            <Route path="/githubassessments/editor" component={GitHubAssessmentsContainer} />
```

Comment:
After making changes it doesn't allow the user to access the editor page unless opening via the missionlisting.
If user is not signed in then it will move back to missionlisting (or 404 not found if the user tries to access using the addressbar).